.gitignore

```diff
@@ -30,3 +30,6 @@ target/
 
 # For the docs generation
 .apm-its
+
+# For the cache java dependencies
+docker/java/spring/.m2
```

Comment:
Why do we have `spring` in the folder name here ? It seems to be a generic maven repository, not really related to spring-framework.

Reply:
.m2 folder empty folder was created in the repo to enable this particular feature, the scripts that run in the CI copy their `~/.m2` folder to `docker/java/spring`, other than that I don't know what you mean :?

Reply:
I mean that not all dependencies are spring dependencies within a local maven repository, thus this `spring` folder will contain things that are not related to `spring` at all, thus the `spring` here is definitely confusing, using `docker/java/maven` instead of `docker/java/spring` would make more sense.

Reply:
I see, no strong opinion, neither I know the reason for that particular folder name, so I'd say that folder name could be renamed in another follow up

Reply:
the folder `docker/java/spring` has this name because it is a spring-boot application, change the name to `docker/java/maven` does not have any sense.